e2e/playwright/code-pane-and-errors.spec.ts

```diff
@@ -235,6 +235,57 @@ extrude001 = extrude(sketch001, length = 5)`
         .first()
     ).toBeVisible()
   })
+
+  test('KCL errors with functions show hints for the entire backtrace', async ({
+    page,
+    homePage,
+    scene,
+    cmdBar,
+    editor,
+  }) => {
+    const u = await getUtils(page)
+
+    // Load the app with empty code.
+    await page.addInitScript(() => {
+      localStorage.setItem('persistCode', '')
+    })
+
+    await page.setBodyDimensions({ width: 1200, height: 500 })
+    await homePage.goToModelingScene()
+    await scene.settled(cmdBar)
+
+    const code = `@settings(defaultLengthUnit = mm)
+
+fn check(@x) {
+  return assert(x, isGreaterThan = 0)
+}
+
+fn middle(@x) {
+  return check(x)
+}
+
+middle(1)
+middle(0)
+`
+    await test.step('Set the code with a KCL error', async () => {
+      await u.openKclCodePanel()
```

Comment:
We're trying to get away from using `getUtils`; you can do this with the `toolbar` fixture like so:
```suggestion
    await toolbar.openPane('code')
```

If you commit this suggestion directly it will fail though: you need to add the `toolbar` fixture above. I wish we could suggest in multiple places sometimes 😞